packages/swarm-docs/src/components/selectExample.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Select } from '@meetup/swarm-components'
+
+const SelectExample = () => {
```

Comment:
We're going to want to demo how to use react to manages the state of `Select` here by using the `value` and `onChange` props.